packages/lesswrong/components/common/NoContent.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Components, registerComponent } from 'meteor/vulcan:core';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+
+const styles = (theme) => ({
+  root: {
+    color: theme.palette.grey[600],
+    margin: theme.spacing.unit*2
+  },
+})
+
+const NoContent = ({children, classes}) => {
+  return <Typography variant='body2' className={classes.root}>
+    {children}
+  </Typography>
+}
+
+registerComponent( 'NoContent', NoContent, withStyles(styles, {name: 'MetaInfo'}))
```

Comment:
Is it possible to export this as an es6 module instead of using `registerComponent`? Avoiding `registerComponent` will make it easier to understand our dependency tree and make things easier if we ever move in the direction of doing code splitting.